include/zephyr/drivers/usb_c/usbc_pd.h

```diff
@@ -53,6 +53,55 @@ extern "C" {
  */
 #define PD_MAX_EXTENDED_MSG_CHUNK_LEN    26
 
+/**
+ * @brief The CapsCounter is used to count the number of Source_Capabilities
+ *	  Messages which have been sent by a Source at power up or after a
+ *	  Hard Reset.
+ *	  See Table 6-63 Counter parameters
+ *	  Parameter Name: CapsCounter
+ */
+#define PD_N_CAPS_COUNT 50
+
+/**
+ * @brief The HardResetCounter is used to retry the Hard Reset whenever there
+ *	  is no response from the remote device (see Section 6.6.6)
+ *	  See Table 6-63 Counter parameters
+ *	  Parameter Name: HardResetCounter
+ */
+#define PD_N_HARD_RESET_COUNT 2
+
+/**
+ * @brief The NoResponseTimer is used by the Policy Engine in a Source to
+ *	  determine that its Port Partner is not responding after a Hard Reset.
+ *	  See Table 6-61 Time Values
+ *	  Parameter Name: NoResponseTimer
+ */
+#define PD_T_NO_RESPONSE_MIN_MS 4500
```

Comment:
Can you group all the parameters from a single table together?  You could also use a doxygen group to collect them together.

```
/**
 * @name USB PD 3.0 Rev 2.0, Table 6-62 Time Values
 * @{
 */
/**
 * @brief The NoResponseTimer is used by the Policy Engine in a Source to
 *	  determine that its Port Partner is not responding after a Hard Reset.
 *	  Parameter Name: NoResponseTimer
 */
#define PD_T_NO_RESPONSE_MIN_MS 4500

/**
 * @brief The NoResponseTimer is used by the Policy Engine in a Source to
 *	  determine that its Port Partner is not responding after a Hard Reset.
 *	  Parameter Name: NoResponseTimer
 */
#define PD_T_NO_RESPONSE_MAX_MS 5500
/* all other table 6-61 parameters ... */
/** @} */
```

You should also list the macros alphabetically to match the PD spec.

Reply:
+1

I'd also suggest updating your spec references. In PD r3.1 v1.6, Time Values, Timers, Counter Parameters, and Counters, are 6-68 - 6-71, respectively.

Reply:
done